lib/chef/rest.rb

```diff
@@ -188,11 +192,6 @@ def follow_redirect
 
     public :create_url
 
-    def http_client(base_url=nil)
-      base_url ||= url
-      BasicClient.new(base_url, :ssl_policy => Chef::HTTP::APISSLPolicy)
-    end
-
```

Comment:
Yeah, `Chef::REST` inherits from `Chef::HTTP`, which has the exact same code as was here.